tests/ui/perfherder/alerts-view/modal_perf_tags_test.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { render, fireEvent, waitFor } from '@testing-library/react';
+
+import TagsModal from '../../../../ui/perfherder/alerts/TagsModal';
+import testPerformanceTags from '../../mock/performance_tags';
```

Comment:
I think you should rebase on latest master & update the mocked performance tags, to include the `improvement` tag.